modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -4,18 +4,28 @@ module AppealsApi::V1
   module DecisionReviews
     module NoticeOfDisagreements
       class EvidenceSubmissionsController < AppealsApi::ApplicationController
+        include SentryLogging
+
+        class EvidenceSubmissionRequestValidatorError < StandardError; end
+
         skip_before_action :authenticate
-        before_action :set_notice_of_disagreement, only: :create
-        before_action :set_submission_attributes, only: :create
+        before_action :nod_id_present?, only: :create
 
         def create
-          upload = VBADocuments::UploadSubmission.create! consumer_name: 'appeals_api_nod_evidence_submission'
-          submission = AppealsApi::EvidenceSubmission.create! @submission_attributes.merge(upload_submission: upload)
+          status, error = AppealsApi::EvidenceSubmissionRequestValidator.new(params[:nod_id],
+                                                                             params['headers']['X-VA-SSN']).call
 
-          render json: submission,
-                 serializer: AppealsApi::EvidenceSubmissionSerializer,
-                 key_transform: :camel_lower,
-                 render_location: true
+          if status == :ok
+            upload = VBADocuments::UploadSubmission.create! consumer_name: 'appeals_api_nod_evidence_submission'
+            submission = AppealsApi::EvidenceSubmission.create! submission_attributes.merge(upload_submission: upload)
+
+            render json: submission,
+                   serializer: AppealsApi::EvidenceSubmissionSerializer,
+                   key_transform: :camel_lower,
+                   render_location: true
+          else
+            render json: { errors: [log_error(error)] }, status: error[:title].to_sym
```

Comment:
I'm not sure we want to rely on the return value of the `log_error` method for rendering the json... Maybe log and then render?